sdk/cognitiveservices/Microsoft.Azure.CS.Vision.CustomVision.Training/src/Microsoft.Azure.CognitiveServices.Vision.CustomVision.Training.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Didn't we agree to shorten the folder names to remove "Microsoft.Azure.CognitiveServices" and let the convention use the project name instead?

Reply:
Yes, I will get that done, but in another PR after this. Thats because I see that there are a few other services that might be affected. There are a few test files in the other services that are not yet named with the Microsoft.Azure convention. I wanted to address all of them together after this is merged.

Reply:
Oh sorry, tests are not included in the <DataPlaneProjectReference>, I will go ahead and make that change now.

Reply:
I have the folders updated now.